### PR TITLE
Dev transparent btn

### PR DESCRIPTION
**Pull request by poom-kitti**
1. Change clickable icon to be in base transparent button instead
a. Pen icon in Wallet 
image
b. Chevron icon in navbar
image
c. Hamburger icon in org side navbar
image

2. Fix api regarding User by changing getting 'events' to 'history'
3. Make page skeleton for org also have min content height

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -15,6 +15,10 @@
     max-width: 570px;
   }
 
+  .content-min-height {
```
Comment (new5558) [inline src/index.css:18]: Isn't this is same as css in pageSkeleton

**Review by new5558 — APPROVED** [review]
LGTM after answer question

File: src/index.css (hunk 1 above)
Comment (poom-kitti) [reply src/index.css:18]: Move from css scoped of pageSkeleton to global krub

Comment (poom-kitti) [reply src/index.css:18]: So that i can use it in other skeleton if needed (for future org skeleton)

**Pull request merged**